routes/markers.js

```diff
@@ -142,9 +142,9 @@ module.exports = function(connection) {
     router.patch('/marker/:id', utils.isAuthenticated, function (req, res, next) {
 
         //If authenticated user is not the owner of the marker or an admin, return 401 (Unauthorized)
-        if(req.session.user.uid != req.body.uid && !req.session.user.admin) return res.send(401);
+        if(req.session.user.uid != req.body.created_by && !req.session.user.admin) return res.send(401);
 
-        if(req.body !== null && !Object.keys(req.body).length) {
+        if(req.body !== null && Object.keys(req.body).length == 1) {
```

Comment:
This line has been changed accordingly

Reply:
This line does not need to change. We need to check the value of `created_by` in the database and compare to that value.